### PR TITLE
Enable overrides per shard

### PR DESCRIPTION
**Pull request by janboll**
Related ticket: https://issues.redhat.com/browse/APPSRE-6100
Related schema change: https://github.com/app-sre/qontract-schemas/pull/216

### CODE REVIEW

File: reconcile/queries.py [hunk 1]
```diff
@@ -219,6 +219,14 @@ def get_credentials_requests():
           }
         }
       }
+      shardSpecOverride{
+        shardingKey {
```
Comment (janboll) [inline reconcile/queries.py:223]: Schema updated, need to change this

**Review by maorfr — APPROVED** [review]
LGTM

is this also testable in https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/test_utils_helm.py?

**Comment by rporres** [comment]
`foo.py` looks like a leftover...

**Comment by janboll** [comment]
My bad, I rebased the branch on master, forgot it will remove the review comments :broom: 
Recent changes are 
- [Add helm specific test](https://github.com/app-sre/qontract-reconcile/pull/2725/commits/eec5dbea083717944e86e93ba0de0f6856c28bc0)
- [Clean up dummy file](https://github.com/app-sre/qontract-reconcile/pull/2725/commits/6095042b627dc3d2515051996aa0cdb5d0365036)

**Pull request merged**